infra/src/test/resources/application-test.yml

```diff
@@ -7,7 +7,6 @@ geoviite:
   scheduling:
     enabled: false
   cache:
-    enabled: false
```

Comment:
Muut profiilit asettavat kakutuksen jo haluttuun arvoon (tässä tapauksessa `enabled`,) ei kosketa siihen enää täällä

Reply:
🤔 Niin taitaa päätyä enabled: true sieltä application.ymlistä, jota ei aktivoida profiililla. Vähän hämäsi kun chekkailin missä profiilissa tuo asetellaan testiajojen yhteydessä.

Reply:
Käytännössä joo näin. Ajattelin että olisi kumminkin fiksumpaa muiluttaa muualta tuleva asetus tänne kuin lukita se taas testiprofiilissa, sillä tuon propsun aiempi kiinnittäminen väärään arvoon oli se syy minkä takia tätä koko refakkia ylipäätään lähdettiin tekemään.